### ISSUE TITLE
Live program from beginning

### ISSUE THREAD

**Issue by 5palo5** [palinek/pvr.sledovanitv.cz#80]
Hello

I have problem with EPG and programs from archive. I can't play program from archive from beginning that is currently in live. Archive works only for programs that already end. 

##### Workaround
I select program from EPG before live and forward it to the end.

##### System Information

* pvr.sledovani.tv Version: 4.5.0
* Kodi Version: 19.3
* OS Version: Linux Arch for Raspberry Pi

thanks

**Comment by palinek** [comment]
That's strange as I don't see such behavior. When playing a program from EPG, which is currently streamed live, the program is normally played, but we just need to rewind it to beginning.

**Comment by 5palo5** [comment]
yes, you're right. I forgot to install binary addon InputStream Adaptive. Now, it's working as you describe. 

Is it possible to fix play archive from beginning for curretly streamd live? (for example: Calulcate diff between EPG and sysdate and rewind it automatically)
